src/gui/qgsmaptip.cpp

```diff
@@ -129,7 +129,6 @@ void QgsMapTip::showMapTip( QgsMapLayer *pLayer,
   bodyStyle = QString(
```

Comment:
Can you change the opportunity to change this to a QStringLiteral?

Reply:
Done